src/main/java/io/permit/sdk/PermitConfig.java

```diff
@@ -194,6 +205,7 @@ public static class Builder {
         // main config vars
         private String token;
         private String pdp = "http://localhost:7766";
+        private String opa = "http://localhost:8181/v1/data/permit";
```

Comment:
opa should be http://localhost:8181
`/v1/data` is the their data api url
and `permit` is just the prefix for our package.
IMO the user permissions rule should be some const or something like that of 
`permit.user_permissions.permissions` ( with replace of `.` with `/` ) and create the url as 
`opaBaseURL + "/v1/data" + userPermissionsPackage.replace(".","/")` ( psuedo code )

Reply:
i didnt do the replace - seems i have to create new string (maybe java not allowes string mutation)
it will looks something like that 
```
       String original = "permit.user_permissions.permissions";

        // Replace all dots with slashes
        String replaced = original.replace(".", "/");

```